File: Python/fizzbuzz.py
def fizzbuzz(multiplo_3: int, multiplo_5: int) -> int:
    if not (isinstance(multiplo_3, int) and isinstance(multiplo_5, int)):
        return (-1)
    
    numero: int 
    numero = 1
    while (numero <= 100):
        if (numero % multiplo_3 == 0 and numero % multiplo_5 == 0):
            print("fizzbuzz")
        elif (numero % multiplo_3 == 0):
            print("fizz")
        elif (numero % multiplo_5 == 0):
            print("buzz")
        else:
            print(numero)
        numero += 1
    return (0)

def main() -> None:
    fizzbuzz(3, 5)

if __name__ == "__main__":
    main()

File: C/fizzbuzz.c
#include <unistd.h>

void print_number(int i) 
{
    if (i > 9)
    {
        print_number(i / 10);
    }
    char c = '0' + i % 10;
    write(1, &c, 1);
}

void fizzbuzz(void)
{
    int i;

    i = 1;
    while (i <= 100)
    {
        if(i % 3 == 0 && i % 5 == 0)
        {
            write(1, "fizzbuzz\n", 10);
        }
        else if(i % 3 == 0)
        {
            write(1, "fizz\n", 6);
        }
        else if(i % 5 == 0)
        {
            write(1, "buzz\n", 6);
        }
        else
        {
            print_number(i);
            write(1, "\n", 1);
        }
        i++;
    }
    
}

int  main(void)
{
    fizzbuzz();
    return(0);
}